sale_order_line_menu/readme/USAGE.rst

```diff
@@ -0,0 +1 @@
+Menu option available at Sales > Orders > Order Lines.
```

Comment:
```suggestion
Menu option available at Sales > Orders > Sale Order Lines.
```

Reply:
I like "Order Lines" because this is also applicable to Quotations.
If not, I would use "Sales Order Lines".
Thoughts?

Reply:
+1 "Order Lines" because as you mentioned it also applicable to Quotations.